libtenzir/src/application.cpp

```diff
@@ -248,7 +247,6 @@ auto make_root_command(std::string_view name) {
   root->add_subcommand(make_count_command());
   root->add_subcommand(make_export_command());
   root->add_subcommand(make_import_command());
-  root->add_subcommand(make_start_command());
```

Comment:
Shouldn't we see it in integration tests if it would break that way? Just from the code in `application.cpp` it looks to me like the behavior of `tenzir-node` would not be changed:

```
    auto cmd = make_start_command();
    cmd->name = "";
    add_root_opts(*cmd);
    return {
      std::move(cmd),
      command::factory{
        {"", start_command},
      },
    };
```

Reply:
Ah, I misunderstood it then. You didn't remove the `start_command` itself, just the call to register it as a subcommand. That makes sense.